jobrunner/reusable_actions.py

```diff
@@ -84,7 +84,7 @@ def handle_reusable_action(run_command):
 
     reusable_action = fetch_reusable_action(image, tag)
     new_run_args = apply_reusable_action(run_args, reusable_action)
-    # When we no longer support Python 3.7 we can just use `shlex.join()` here
+    # we can't use `shlex.join()` here because it fails on Windows
```

Comment:
That's a pain.

Reply:
The trackback makes it look like that failure was under 3.7:
```
C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\shutil.py:400: in _rmtree_unsafe
```

Are you sure `shlex.join()` was the culprit here?

Reply:
Well now I'm not 😰

Reply:
Yep, looks like I just messed it up, working in the latest actions run.